scripts/windows/package.ps1

```diff
@@ -0,0 +1,40 @@
+#Requires -Version 5.0
+$ErrorActionPreference = "Stop"
+
+$DIR_PATH = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$SRC_PATH = (Resolve-Path "$DIR_PATH\..\..").Path
+cd $SRC_PATH\package\windows
+
+$ARCH = $env:ARCH
+$TAG = $env:TAG
+$REPO = $env:REPO
+$IMAGE = "$REPO/fleet-agent:$TAG"
+$HOST_RELEASE_ID = $env:HOST_RELEASE_ID
```

Comment:
you grabbed this host release id stuff from rancher-agent build and you dont really use it. it's an old if/else to do hyperv builds for windows versions not matching and trying to run it in hyperv. i dont think this stuff ever worked and you can get rid of it and do the docker build normally and ignore the stuff in the else down below.

Reply:
Didn't even need this file